spec/unit/target/pod_target_spec.rb

```diff
@@ -662,6 +662,39 @@ module Pod
         end
       end
 
+      describe 'resource and framework paths' do
```

Comment:
moved these to its own describe method, they should not have been under "test spec support"